boards/arm/colibri_imx7d_m4/colibri_imx7d_m4.dts

```diff
@@ -30,17 +30,25 @@
 
 	leds {
 		compatible = "gpio-leds";
+		#address-cells = <1>;
+		#size-cells = <0>;
+
 		green_led: led@0 {
 			gpios = <&gpio1 2 GPIO_INT_ACTIVE_LOW>;
 			label = "User LED1";
+			reg = <0>;
```

Comment:
FYI, for this particular warning type, I proposed to enable a waiver (cf https://github.com/zephyrproject-rtos/zephyr/issues/7564). Rational is I don't think adding a reg makes much sense here (while removing @0 would have bad side effects).

Reply:
I agree w/Erwan, we shouldn't add reg here.

Reply:
Should I still add #address-cells and #size-cells? If I do, I get a different warning:
```
frdm_k64f.dts_compiled: Warning (avoid_unnecessary_addr_size): unnecessary #address-cells/#size-cells without "ranges" or child "reg" property in /leds
frdm_k64f.dts_compiled: Warning (avoid_unnecessary_addr_size): unnecessary #address-cells/#size-cells without "ranges" or child "reg" property in /gpio_keys
```